api/public/v2/repo/views.py

```diff
@@ -44,7 +44,16 @@ class RepositoryViewSet(
     def get_queryset(self):
         return super().get_queryset().with_recent_coverage()
 
-    @extend_schema(summary="Repository list")
+    @extend_schema(
+        summary="Repository list",
+        parameters=[
+            OpenApiParameter(
+                name="names",
+                type={"type": "array", "items": "string"},
+                explode=True,
```

Comment:
Added explode after reading about it here: https://swagger.io/docs/specification/describing-parameters/